frontend/lib/src/components/elements/LinkButton/LinkButton.tsx

```diff
@@ -68,6 +74,12 @@ function LinkButton(props: Readonly<Props>): ReactElement {
           rel="noreferrer"
           aria-disabled={disabled}
         >
+          <DynamicIcon
+            iconValue={element.icon}
+            color={colors.bodyText}
+            size={isMaterialIcon ? "lg" : "base"}
+            margin={isMaterialIcon ? "0 sm 0 0" : "0 md 0 0"}
+          />
```

Comment:
Should only render this component if an icon is passed:
```javascript
          {element.icon && (
            <DynamicIcon
              size={isMaterialIcon ? "lg" : "base"}
              margin={isMaterialIcon ? "0 sm 0 0" : "0 md 0 0"}
              color={colors.bodyText}
              iconValue={element.icon}
            />
          )}
```

Reply:
Great catch, updating!